### PR TITLE
OP-875 link bill admission

### PR DESCRIPTION
**Pull request by mwithi**
See OP-875.
Paired with https://github.com/informatici/openhospital-gui/pull/1332.

### CODE REVIEW

**Review by dbmalkovsky — APPROVED** [review]
LGTM if the `step_XX_` file name is corrected before the merge

**Review by emecas — APPROVED** [review]

**Pull request merged**
